go-controller/pkg/node/gateway_init.go

```diff
@@ -143,8 +143,12 @@ func (n *OvnNode) initGateway(subnets []*net.IPNet, nodeAnnotator kube.Annotator
 		return err
 	}
 
-	// Wait for gateway resources to be created by the master
-	waiter.AddWait(gatewayReady, prFn)
+	// Wait for gateway resources to be created by the master if DisableSNATMultipleGWs is not set,
+	// as that option does not add default SNAT rules on the GR and the gatewayReady function checks
+	// those default NAT rules are present
+	if !config.Gateway.DisableSNATMultipleGWs {
+		waiter.AddWait(gatewayReady, prFn)
```

Comment:
@rcarrillocruz @trozet instead of disabling this check, let's just find another check. There's nothing magic about the lr-nat-add check, it's just "this is the last thing the master does for gateways that's easy to check". It could be a route or anything else.

Reply:
@rcarrillocruz this can wait for a follow-up

Reply:
ack